modules/utilityFunction.js

```diff
@@ -0,0 +1,127 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export function loadBookslist(bookInstanceFromBookStore) {
+  function loopingBook(book) {
+    return `<div id="${book.id}" class="book-card">
+                  <p class="book-details">
+                      "${book.title}" 
+                      by 
+                      ${book.author}
+                  </p>
+                  <button class="remove-button" data-book="${book.id}">Remove</button>
+              </div>`;
+  }
+
+  const bookContainer = document.querySelector('.books-container');
+  bookContainer.innerHTML = bookInstanceFromBookStore.store.map((book) => loopingBook(book)).join('');
+
+  // remove button
+
+  const removeButtons = document.querySelectorAll('.remove-button');
+  removeButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      bookInstanceFromBookStore.remove(id);
+      loadBookslist(bookInstanceFromBookStore);
+      localStorage.setItem('bookCollection', JSON.stringify(bookInstanceFromBookStore.store));
+    });
+  });
+}
+
+export function loadContacts(contactInstanceFromContact) {
+  function loopingContacts(contact) {
+    return `<div class="contact-card">
+                <div class="contact-image ${contact.classImage}"></div>
+                <div class="contact-name text-center">${contact.name}</div>
+                <div class="contact-details">
+                  <a class="contact-link" target="_blank" href="${contact.githubLink}">
+                    <i class="fab fa-github-square fa-fw fa-2xl"></i>
+                  </a>
+                  <a class="contact-link" target="_blank" href="${contact.linkedLink}">
+                    <i class="fab fa-linkedin fa-fw fa-2xl"></i>
+                  </a>
+                  <a class="contact-link" target="_blank" href="${contact.instagramLink}">
+                    <i class="fab fa-instagram-square fa-fw fa-2xl"></i>
+                  </a>
+                </div>
+              </div>`;
+  }
+
+  const contactWrapper = document.querySelector('.contact-wrapper');
+
+  contactWrapper.innerHTML = contactInstanceFromContact.profile.map((contact) => loopingContacts(contact)).join('');
+}
+
+export function addBookButton(bookInstanceFromBookStore) {
+  const titleInput = document.querySelector('#title-input');
+  const authorInput = document.querySelector('#author-input');
```

Comment:
- [ ] The keyword `function` is depreciated in the new javascript ES6 syntax and is replaced by the arrow functions. So I would suggest you to use  `arrow functions` instead of function keyword as well. 

**Example snippet:**

```js
export const addBookButton = (bookInstanceFromBookStore)=> {
...
}

```